### DISCUSSION TITLE
Run NextFlow as JAR library

### DISCUSSION THREAD

**Discussion by AcaDemIQ** [nextflow-io/nextflow#1917] [Q&A]
Hello, we need to run nextflow platform under Java + Spring context, not using CLI.

Can you describe how we can run it?)

**Comment by abhi18av** [comment]
Hi @AcaDemIQ , I'm also curious about this thing.

Have you been able to get it working?

If not, perhaps the `nextflow launcher` script might be of help 
https://github.com/nextflow-io/nextflow/blob/f7f7b44bb84b5e7679ef48ea23544a8ab70366e8/nextflow#L269

**Reply by AcaDemIQ** [reply]
Oh, `nextflow` is a just BASH script :)
Okey, I'll try to research this runnable script
